### DISCUSSION TITLE
Transcluded block text doesn't include link

### DISCUSSION THREAD

**Discussion by ahmedelq** [excalidraw/excalidraw#7960] [Q&A] [answered]
Link doesn't appear correctly for a transcluded text block (picture below)
Is this a bug or am I missing a setting? 

![image](https://github.com/excalidraw/excalidraw/assets/11134754/6e961330-bb7e-4bb2-b459-7b32c9d7a0a9)

**Comment by Jacobvdb** [comment, marked as answer]
It seems that hyperlinks on embedded presentations do not work. [https://bkper.com/slides/basic-concepts/groups/](https://bkper.com/slides/basic-concepts/groups/) slide 6

**Reply by Mrazator** [reply]
Just to elaborate further, this looks like an markdown / obisidan link, if it's related directly to obsidian, please report it here https://github.com/zsviczian/obsidian-excalidraw-plugin (the plugin is not maintained by us).

For in text links, we might support them one day as part of the rich text.

---

Note that it's possible to create links between elements.

https://x.com/excalidraw/status/1861799712069767586

**Reply by Jacobvdb** [reply]
thank you
